users/discord/App.js

```diff
@@ -46,23 +49,45 @@ const handleUserSelected = (e) => {
     showUser = usersData[activeTab]?.findIndex(
       (user) => user.id === selectedUserId,
     );
+    searchTerm = '';
     rerender(App(), window['root']);
   }
 };
+const users = usersData[activeTab] ?? [];
 
-export const App = () => {
-  const users = usersData[activeTab] ?? [];
+const filteredUsers = users.filter(
+  (user) =>
+    user.github_display_name &&
+    user.github_display_name.toLowerCase().includes(searchTerm.toLowerCase()),
+);
```

Comment:
What is the search key?

Reply:
on button click the search term gets appended in url and page Rerenders and the search term is fetched from there and then users are  filtered on basis of that search term.

Reply:
my question is based on which field is the search being performed here.
Is it based on a username? first name, last name, GitHub username?
Which field is used for the search?

Reply:
Based on github displayname. Which for all users were there actual names.